cucumber/features/steps/visibility.rb

```diff
@@ -34,6 +34,9 @@ def query_for_automatable_elements(uiquery)
 end
 
 Then(/^the status bar is visible and sometimes hitable$/) do
+  # Skip this step since StatusBar was deprecated on iOS 13+
+  next if ios_gte?("13.0")
```

Comment:
The tests are explanatory - they help us understand what has happened between versions and to make statements like:  "Sometimes on iPhone X devices (not simulators), the status bar is not hitable and not visible".

Let's capture that we know the status bar does not exist on iOS 13.

We will want to know if the status bar is present on iPadOS (for example).  Or maybe the status bar is present in iOS 14.  Or on iPhone SE phone.  Or on simulators.

if ios_gte?("13.0")
 wait_for_no_view({type: "StatusBar", all: true})
elsif iphone_x? && !device_info["simulator"]
 ...
end
```

Reply:
That makes sense